concepticondata/conceptlists/BirketSmith-1928-510.tsv

```diff
@@ -0,0 +1,511 @@
+BirketSmith-1928-510-108	108	Pike	2967	PIKE (FISH)	Animals
+BirketSmith-1928-510-109	109	Ptarmigan, rock			Animals
+BirketSmith-1928-510-110	110	Ptarmigan, willow			Animals
+BirketSmith-1928-510-111	111	Raven	2543	RAVEN	Animals
+BirketSmith-1928-510-112	112	Salmon	944	SALMON	Animals
+BirketSmith-1928-510-113	113	Sea scorpion			Animals
+BirketSmith-1928-510-114	114	Seal, hooded			Animals
+BirketSmith-1928-510-115	115	Seal, saddleback			Animals
+BirketSmith-1928-510-116	116	Seal, saddleback, young			Animals
+BirketSmith-1928-510-117	117	Seal, ringed			Animals
+BirketSmith-1928-510-118	118	Seal, ringed (stinking)			Animals
+BirketSmith-1928-510-119	119	Seal, spotted			Animals
+BirketSmith-1928-510-120	120	Seal, bearded			Animals
+BirketSmith-1928-510-121	121	Seal, beareded (young)			Animals
+BirketSmith-1928-510-122	122	Shark, Greenland			Animals
+BirketSmith-1928-510-123	123	Spider	843	SPIDER	Animals
+BirketSmith-1928-510-124	124	Spider, earth			Animals
+BirketSmith-1928-510-125	125	Starfish	3032	STARFISH	Animals
+BirketSmith-1928-510-126	126	Swan	2837	SWAN	Animals
+BirketSmith-1928-510-127	127	Tern			Animals
+BirketSmith-1928-510-128	128	Trout	795	TROUT	Animals
+BirketSmith-1928-510-129	129	Trout, freshwater			Animals
+BirketSmith-1928-510-130	130	Walrus			Animals
+BirketSmith-1928-510-131	131	Whale, right			Animals
+BirketSmith-1928-510-132	132	Whale, sperm			Animals
+BirketSmith-1928-510-133	133	Whale, humpback			Animals
+BirketSmith-1928-510-134	134	Whale, killer			Animals
+BirketSmith-1928-510-135	135	Whale, white			Animals
+BirketSmith-1928-510-136	136	Wolf	522	WOLF	Animals
+BirketSmith-1928-510-137	137	Wolverene			Animals
+BirketSmith-1928-510-138	138	Birch, dwarf			Plants
+BirketSmith-1928-510-139	139	Cassiope			Plants
+BirketSmith-1928-510-140	140	Cowberry			Plants
+BirketSmith-1928-510-141	141	Crowberry			Plants
+BirketSmith-1928-510-142	142	Grass, stalk of			Plants
+BirketSmith-1928-510-143	143	Moss, lamp			Plants
+BirketSmith-1928-510-144	144	Moss, reindeer			Plants
+BirketSmith-1928-510-145	145	Whortleberry	2809	BLUEBERRY	Plants
+BirketSmith-1928-510-146	146	Willow	818	WILLOW	Plants
+BirketSmith-1928-510-147	147	Ankle	803	ANKLE	Man and the Human Body
+BirketSmith-1928-510-148	148	Anus	1935	ANUS	Man and the Human Body
+BirketSmith-1928-510-149	149	Arm	1673	ARM	Man and the Human Body
+BirketSmith-1928-510-150	150	Arm, upper	431	UPPER ARM	Man and the Human Body
+BirketSmith-1928-510-151	151	Beard	678	BEARD	Man and the Human Body
+BirketSmith-1928-510-152	152	Blood	946	BLOOD	Man and the Human Body
+BirketSmith-1928-510-153	153	Bone	1394	BONE	Man and the Human Body
+BirketSmith-1928-510-154	154	Breast (mamma)	1402	BREAST	Man and the Human Body
+BirketSmith-1928-510-155	155	Brow	1173	EYEBROW	Man and the Human Body
+BirketSmith-1928-510-156	156	Cheak	1730	CHEEK	Man and the Human Body
+BirketSmith-1928-510-157	157	Chest	1592	CHEST	Man and the Human Body
+BirketSmith-1928-510-158	158	Child (able to walk)			Man and the Human Body
```

Comment:
I was hesitant because of the 'able to walk' part. However, since this is most likely used to differentiate from 'baby' mapping tho CHILD makes sense.